app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,85 @@
+class TweetsController < ApplicationController
+    
+    get '/tweets' do 
+        @tweets = Tweet.all
+        if logged_in?
+            erb :'/tweets/tweets'
+        else
+            redirect to '/login'
+        end
+    end
+
+    get '/tweets/new' do 
+        if logged_in?
+            erb :'/tweets/create_tweet'
+        else
+            redirect to '/login'
+        end
+    end
+
+    get '/tweets/:id' do 
+        if logged_in?
+            @tweet = Tweet.find_by_id(params[:id])
+            erb :'/tweets/show_tweet'
+        else
+            redirect to '/login'
+        end
+    end
+
+    post '/tweets' do
+        if logged_in?
+            if !params[:content].empty? 
+                @tweet = Tweet.create(content: params[:content])
+                current_user.tweets << @tweet #current_user.tweet.build(content: params[:content])
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```

Reply:
Your commented out attempt actually looks great! Looks like all you're missing is an 's': ```
@tweet = current_user.tweets.create(content: params[:content])
```
Because a User has_many Tweets, ActiveRecord will give any user a method `tweets` to view those tweets belonging to the user. It is this method that `#build` is built from.